scripts/kind-e2e/README.md

```diff
@@ -0,0 +1,111 @@
+# E2E environment
+
+The e2e environment can be used for local testing, development, and CI purposes.
+
+E2E environment consists of:
+- 3 k8s clusters deployed with [kind].
```

Comment:
it mentions three clusters but then there are only two configs and the `for` that sets the envs also only brings up two

Reply:
Readme fixed.